lib/Bio/EnsEMBL/Tark/Schema/Result/Gene.pm

```diff
@@ -163,7 +163,7 @@ __PACKAGE__->add_columns(
   {
     data_type => "integer",
     extra => { unsigned => 1 },
-    is_foreign_key => 1,
+    # is_foreign_key => 1,
```

Comment:
You've just surrendered the ability to regenerate the DBIC schema with this change. That means all schema synchronisation with Tark is now manual.